ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -411,6 +425,25 @@ async fn process_streaming_response(
                 break;
             },
         }
+
+        // Check if the file store has been updated.
+        if last_file_update_check_timestamp.elapsed().as_secs() >= GCS_LOOKUP_FREQUENCY_IN_SECS {
```

Comment:
This logic doesn't make sense. We run this check once every 60 seconds but we only sleep 1 second? It's basically as if this logic doesn't exist lol. I'll just fix in a different PR since this one is already merged into the other branch.